# origin: Debian (herbert)
# cset: n/a
# inclusion: not suitable for mainline
# description: set ?time to 0 for cramfs
# revision date: 2004-09-03

diff -urN kernel-source-2.4.22-2.4.22.orig/fs/cramfs/inode.c kernel-source-2.4.22-2.4.22/fs/cramfs/inode.c
--- kernel-source-2.4.22-2.4.22.orig/fs/cramfs/inode.c	2002-08-03 10:39:45.000000000 +1000
+++ kernel-source-2.4.22-2.4.22/fs/cramfs/inode.c	2002-08-30 20:40:16.000000000 +1000
@@ -54,6 +54,9 @@
 		inode->i_blksize = PAGE_CACHE_SIZE;
 		inode->i_gid = cramfs_inode->gid;
 		inode->i_ino = CRAMINO(cramfs_inode);
+		inode->i_atime = 0;
+		inode->i_mtime = 0;
+		inode->i_ctime = 0;
 		/* inode->i_nlink is left 1 - arguably wrong for directories,
 		   but it's the best we can do without reading the directory
 	           contents.  1 yields the right result in GNU find, even
